pkg/types/installconfig.go

```diff
@@ -31,6 +32,7 @@ var (
 	// platform names in alphabetical order. This is the list of
 	// platforms presented to the user in the interactive wizard.
 	PlatformNames = []string{
+		alibabacloud.Name,
```

Comment:
This is good for now and we need it here to be able to manually test the installer binary, but before getting this PR merged we'll need to move Alibaba to the `HiddenPlatformNames` slice [here](https://github.com/openshift/installer/blob/master/pkg/types/installconfig.go#L45). That'll make it hidden for end users while we're not yet GA.

Reply:
Sounds good